.devcontainer/devcontainer.json

```
@@ -24,7 +24,8 @@
 
     "features": {
         // Use the GitHub CLI to easily push and create PRs from terminal.
-        "ghcr.io/devcontainers/features/github-cli:1": {}
+        "ghcr.io/devcontainers/features/github-cli:1": {},
+        "powershell": "latest"
```

Where'd you see this document as such? According to https://github.com/devcontainers/features/tree/main/src/powershell, it should be:
```suggestion
        "ghcr.io/devcontainers/features/powershell:1": {}
```

---

Found it at https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/powershell.md however if I click through I get to https://github.com/devcontainers/features/tree/main/src/powershell which is what you are suggesting.